docs/content/getting-started/quickstart.mdx

```diff
@@ -0,0 +1,198 @@
+
+1. Clone the Dagster Quickstart repository by executing:
+
+   ```bash
+   git clone https://github.com/dagster-io/dagster-quickstart && cd dagster-quickstart
+   ```
+
+2. Install the necessary dependencies using the following command:
+
+   We use `-e` to install dependencies in ["editable mode"](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs). This allows changes to be automatically applied when we modify code.
+
+   ```bash
+   pip install -e ".[dev]"
+   ```
+
+3. Run the project!
+
+   ```bash
+   dagster dev
+   ```
+
+4. Navigate to <a href="localhost:3000">localhost:3000</a> in your web browser.
+
+5. **Success!**
+
+</TabItem>
+<TabItem name="Option 2: Using GitHub Codespaces">
+
+### Option 2: Using GitHub Codespaces
+
+1. Fork the [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) repository
+
+2. Select **Create codespace on main** from the **Code** dropdown menu.
+
+<Image
+width={400}
+height={400}
+alt="Create codespace"
+src="/images/getting-started/quickstart/github-codespace-create.png"
+/>
+
+3. After the codespace loads, start Dagster by running `dagster dev` in the terminal:
+
+   ```bash
+   dagster dev
+   ```
+
+4. Click **Open in Browser** when prompted.
+
+<Image
+width={400}
+height={300}
+alt="Codespace Open In Browser"
+src="/images/getting-started/quickstart/github-codespace-open-in-browser.png"
+/>
+
+5. **Success!**
+
+</TabItem>
+</TabGroup>
+
+## Navigating the User Interface
+
+You should now have a running instance of Dagster! From here, we can run our data pipeline.
+
+To run the pipeline, click the **Materialize All** button in the top right. In Dagster, _materialization_ refers to executing the code associated with an asset to produce an output.
+
+<Image
+alt="HackerNews assets in Dagster's Asset Graph, unmaterialized"
+src="/images/getting-started/quickstart/quickstart-unmaterialized.png"
+width={2000}
+height={816}
+/>
+
+Congratulations! You have successfully materialized two Dagster assets:
+
+<Image
+alt="HackerNews asset graph"
+src="/images/getting-started/quickstart/quickstart.png"
+width={2000}
+height={1956}
+/>
+
+But wait - there's more. Because the `hackernews_top_stories` asset returned some `metadata`, you can view the metadata right in the UI:
+
+1. Click the asset
+2. In the sidebar, click the **Show Markdown** link in the **Materialization in Last Run** section. This opens a preview of the pipeline result, allowing you to view the top 10 HackerNews stories:
+
+<Image
+alt="Markdown preview of HackerNews top 10 stories"
+src="/images/getting-started/quickstart/hn-preview.png"
+width={2000}
+height={1754}
+/>
+
+## Understanding the Code
+
+The quickstart project defines two **Assets** using the `@asset` decorator:
```

Comment:
This is `Quickstart project` elsewhere in the guide. We should be consistent about how we write it.